binaries/ofs.uio/uio/ofs_uio.py

```diff
@@ -687,6 +737,26 @@ def main():
                     print('MailboxDump({}): {}'
                           .format(hex(addr), hex(value)))
 
+        elif args.mailbox_json is not None:
+            with open(args.mailbox_json, 'r') as f:
+                try:
+                    data = json.load(f)
```

Comment:
The JSON file is being parsed twice. Once here and once in verify_json.  Can you optimize it so that it loads the JSON only once?

Reply:
fixed